tests/test_pedigree.py

```diff
@@ -174,3 +175,153 @@ def test_pedigree_sanity_checks(self):
 
         with pytest.raises(ValueError):
             ped.get_times(individual=ped.individual)
+
+
+def simulate_pedigree(
+    n_founders=8,
+    n_children_prob=None,
+    n_generations=3,
+    random_sex=True,
```

Comment:
I'm not sure we need the non-random sex option, do we? Probably isn't going to matter for the simulations we'll be doing (we're not going to take sex into account yet) so I think it's just extraneous complexity.

Reply:
I needed it for `test_grandparents`, so that there are enough males and females in the second generation to make the third generation. Should I simplify the code even further and not pair mates by sex?

Reply:
It might be worth it - it's probably a good thing if we're producing super weird tangled up pedigrees. I guess the first pass at this could be the simplest possible pedigree simulator, which makes no attempt at all at realism.

Reply:
ok, I'll change it to ignore sex when pairing up mates